scripts/update/sonarr.sh

```diff
@@ -1,112 +1,104 @@
 #!/bin/bash
-
 if [[ -f /install/.sonarr.lock ]]; then
-    #Check if mono needs an update
-    . /etc/swizzin/sources/functions/mono
-    mono_repo_update
-    systemctl try-restart sonarr
+    #shellcheck source=sources/functions/utils
+    . /etc/swizzin/sources/functions/sonarr
 
-    # Update Sonarr nginx config to bakerboy specs
-    if grep -q "8989/sonarr" /etc/nginx/apps/sonarr.conf; then
-        echo_progress_start "Upgrading nginx config for Sonarr"
-        bash /etc/swizzin/scripts/nginx/sonarr.sh
-        systemctl reload nginx -q
-        echo_progress_done "Nginx config for Sonarr upgraded"
-    fi
-fi
+    #Move sonarr installs to .net
+    if grep -q "ExecStart=/usr/bin/mono" /etc/systemd/system/sonarr.service; then
```

Comment:
swizzin base is /opt for sonarr not the package, correct?

probably should ensure sonarr is not installed via apt to be safe

Reply:
yeah, you're right-- should add case for an apt install. :/

Reply:
@bakerboy448 can you take a look at this pr and check it over?